blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -117,6 +130,20 @@ const MediumPromo = ({ customFields }) => {
     <>
       <article className="container-fluid medium-promo">
         <div className={`medium-promo-wrapper ${customFields.showImage ? 'md-promo-image' : ''}`}>
+          {customFields.headlinePosition === 'above'
```

Comment:
Can you fix existing tests and add logic for using headlinePosition? if you're unsure about testing, react testing library has incredible docs. If you want as well, I'd be happy to screenshare. My slack is @jackhowa as well

Reply:
rtl docs https://testing-library.com/docs/react-testing-library/intro